Source/WebCore/html/HTMLTemplateElement.cpp

```diff
@@ -155,10 +155,10 @@ void HTMLTemplateElement::attachAsDeclarativeShadowRootIfNeeded(Element& host)
         return;
 
     auto delegatesFocus = hasAttributeWithoutSynchronization(HTMLNames::shadowrootdelegatesfocusAttr);
-
     auto clonable = hasAttributeWithoutSynchronization(HTMLNames::shadowrootclonableAttr);
+    auto serializable = hasAttributeWithoutSynchronization(HTMLNames::shadowrootserializableAttr);
 
-    auto exceptionOrShadowRoot = host.attachDeclarativeShadow(*mode, delegatesFocus, clonable);
+    auto exceptionOrShadowRoot = host.attachDeclarativeShadow(*mode, delegatesFocus, clonable, serializable);
```

Comment:
Can we use enum here instead of two booleans? As is, there is no type check done on either value so if someone swaps the order of booleans, the compiler wouldn't catch it.